remember_test.go

```diff
@@ -0,0 +1,47 @@
+package remember_test
+
+import (
+	"remember"
+	"testing"
+)
+
+func TestMemorise(t *testing.T) {
+	memory, err := remember.NewMemory(
```

Comment:
This test can also be run in parallel with `t.Parallel()`, since it has its own `Memory` and input/output.